homeassistant/components/lifx/light.py

```diff
@@ -237,6 +258,21 @@ async def set_state(self, **kwargs: Any) -> None:
         # Update when the transition starts and ends
         await self.update_during_transition(fade)
 
+    async def set_hev_cycle_state(
+        self, power: bool, duration: int | None = None
+    ) -> None:
+        """Set the state of the HEV LEDs on a LIFX Clean bulb."""
+        if lifx_features(self.bulb)["hev"] is False:
```

Comment:
```suggestion
        if not lifx_features(self.bulb)["hev"]
```

Do we need an explicit `False` check here?

Reply:
It's there because I find "if x is True/False" syntax far easier and quicker to parse than "if not X". Is there a particular reason why "If not X" is better in this context?

Reply:
For me, it is less to read, but I agree it doesn't make much different though.